cmd/catalyst-node/catalyst-node.go

```diff
@@ -348,3 +359,66 @@ func main() {
 
 	os.Exit(exitCode)
 }
+
+func startCatalystWebServer(httpAddr string) {
+	http.Handle("/hls/", redirectHlsHandler())
+	glog.Infof("HTTP server listening on %s", httpAddr)
+	http.ListenAndServe(httpAddr, nil)
```

Comment:
Should handle an error on the listener, yeah? Looks like this is a common pattern:
```suggestion
	glog.Fatalf(http.ListenAndServe(httpAddr, nil))
```

Reply:
(But I didn't actually check that it doesn't crash the process with `FATAL ERROR: nil` so you should probably check that 😃)